src/clients/__tests__/oidc-react.test.ts

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/camelcase */
+import { UserManager } from 'oidc-client';
+import {
+  EventListeners,
+  configureClient,
+  createEventListeners
+} from '../__mocks__';
+import { ClientStatus, Client, ClientEvent, ClientError } from '../index';
+import { createOidcClient } from '../oidc-react';
+import { mockMutatorGetterOidc } from '../__mocks__/oidc-react-mock';
+
+// Allows for awaiting promises without try-catch-blocks
+// Inspired by https://blog.grossman.io/how-to-write-async-await-without-try-catch-blocks-in-javascript/
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+async function to(promise: Promise<unknown>) {
+  try {
+    const data = await promise;
+    return [null, data];
+  } catch (err) {
+    return [err];
+  }
+}
```

Comment:
This code seems to be repeated in multiple files. Could it be extracted to a single place somewhere?

Reply:
Ah, forgot about this common function. I imported a npm package that does the same.